mercury_app/migrations/0001_initial.py

```diff
@@ -22,7 +22,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=128)),
                 ('description', models.CharField(blank=True, max_length=1024, null=True)),
-                ('eb_event_id', models.CharField(default=0, max_length=40, unique=True)),
```

Comment:
Why do you keep changing initial migration instead of auto creating new ones? just being curios..